shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -111,22 +111,26 @@ - (instancetype)initWithEngine:(FlutterEngine*)engine
   return self;
 }
 
+- (void)sharedSetupWithProject:(nullable FlutterDartProject*)project {
+  _viewOpaque = YES;
+  _weakFactory = std::make_unique<fml::WeakPtrFactory<FlutterViewController>>(self);
+  _engine.reset([[FlutterEngine alloc] initWithName:@"io.flutter"
```

Comment:
Isn't `_engine` `nil` here?

Reply:
_engine is a smart pointer, it represents nil until this reset function is called on it.